src/collections/_documentation/workflow/integrations/integration-platform/ui-components.md

```diff
@@ -188,18 +210,18 @@ This feature allows the developer to insert a link within a stack trace frame. T
 
 **Attributes:**
 
-- uri - (Required) The link destination. We will automatically add the following query params to the link.
-    - `installationId`: Your integration's installation ID (helps you determine the requesting Sentry org)
-    - `projectSlug`: slug of the project the issue belongs to
-    - `filename`: full path of the stack frame file
-    - `lineNo`: line number of the stack trace in the file
+- `uri` - (Required) The link destination. We will automatically add the following query params to the link.
```

Comment:
```suggestion
- `uri` - (Required) The link destination. Sentry will automatically add the following query params to the link.
```

Reply:
Welp, this is just one of those cases where we gotta keep the periods to identify the separate ideas.